tsdb/chunkenc/varbit_buckets.go

```diff
@@ -55,19 +74,19 @@ func putInt64VBBucket(b *bstream, val int64) {
 	case val == 0:
 		b.writeBit(zero)
 	case bitRange(val, 3): // -3 <= val <= 4
-		b.writeBits(0x02, 2) // '10'
+		b.writeBits(0b10, 2)
 		b.writeBits(uint64(val), 3)
 	case bitRange(val, 6): // -31 <= val <= 32
-		b.writeBits(0x06, 3) // '110'
+		b.writeBits(0b110, 3)
 		b.writeBits(uint64(val), 6)
 	case bitRange(val, 9): // -255 <= val <= 256
-		b.writeBits(0x0e, 4) // '1110'
+		b.writeBits(0b1110, 4)
 		b.writeBits(uint64(val), 9)
 	case bitRange(val, 12): // -2047 <= val <= 2048
-		b.writeBits(0x1e, 5) // '11110'
+		b.writeBits(0b11110, 5)
 		b.writeBits(uint64(val), 12)
 	default:
-		b.writeBits(0x3e, 5) // '11111'
```

Comment:
I spent some time debugging why it was not working only to narrow down to this where 0x3e is actually `111110` and not `11111`. So I changed all `0x` representation to `0b` for easy of read.

Reply:
We have switched to 0b in 40126a849466847269b555d6b0871b9f5c6f98a9, we should change verywhere it makes sense.

Reply:
Good to know! This is a copy-paste code (a different file) which was not in sync with the xor code :)